From d6d87288082bc4707fae54903709fc2744439bd1 Mon Sep 17 00:00:00 2001
From: Baruch Siach <baruch@tkos.co.il>
Date: Tue, 18 Jul 2017 15:21:18 +0300
Subject: [PATCH] git: use BR2_KERNEL_MIRROR as download site

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 package/git/git.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/git/git.mk b/package/git/git.mk
index 9cc88dd726..18f3f17d84 100644
--- a/package/git/git.mk
+++ b/package/git/git.mk
@@ -6,7 +6,7 @@
 
 GIT_VERSION = 2.13.0
 GIT_SOURCE = git-$(GIT_VERSION).tar.xz
-GIT_SITE = https://www.kernel.org/pub/software/scm/git
+GIT_SITE = $(BR2_KERNEL_MIRROR)/software/scm/git
 GIT_LICENSE = GPL-2.0, LGPL-2.1+
 GIT_LICENSE_FILES = COPYING LGPL-2.1
 GIT_DEPENDENCIES = zlib host-gettext
-- 
2.14.1

