From b1e0b56f96edf8affa2ac037143b122772cc9801 Mon Sep 17 00:00:00 2001
From: "A.I" <ailis@paw.zone>
Date: Sat, 8 Jul 2017 13:29:56 -0700
Subject: [PATCH] fix WARNING: unsafe header/library path

---
 criu/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/criu/Makefile b/criu/Makefile
index b2c6633..2e4d6d6 100644
--- a/criu/Makefile
+++ b/criu/Makefile
@@ -22,7 +22,6 @@ ccflags-y		+= -iquote include
 ccflags-y		+= -iquote images
 ccflags-y		+= -iquote $(ARCH_DIR)/include
 ccflags-y		+= -iquote .
-ccflags-y		+= -I/usr/include/libnl3
 ccflags-y		+= $(COMPEL_UAPI_INCLUDES)
 
 export ccflags-y
-- 
2.10.1

