From 291529eec5a5835886b27263cbb2fa415fdefef0 Mon Sep 17 00:00:00 2001
From: "A.I" <ailis@paw.zone>
Date: Sat, 8 Jul 2017 13:29:56 -0700
Subject: [PATCH] fix WARNING: unsafe header/library path

---
 criu/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/criu/Makefile b/criu/Makefile
index b3b09ce7a..1872d87d9 100644
--- a/criu/Makefile
+++ b/criu/Makefile
@@ -20,7 +20,6 @@ ccflags-y		+= -iquote include
 ccflags-y		+= -iquote images
 ccflags-y		+= -iquote $(ARCH_DIR)/include
 ccflags-y		+= -iquote .
-ccflags-y		+= -I/usr/include/libnl3
 ccflags-y		+= $(COMPEL_UAPI_INCLUDES)
 
 export ccflags-y
-- 
2.14.1

