From 9964ce0c2fb494fed693782f9e44c9185eb34949 Mon Sep 17 00:00:00 2001
From: "A.I" <ailis@paw.zone>
Date: Sat, 8 Jul 2017 13:29:56 -0700
Subject: [PATCH] fix WARNING: unsafe header/library path

---
 criu/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/criu/Makefile b/criu/Makefile
index bcfb71622..409dfbff2 100644
--- a/criu/Makefile
+++ b/criu/Makefile
@@ -26,7 +26,6 @@ ccflags-y		+= -iquote include
 ccflags-y		+= -iquote images
 ccflags-y		+= -iquote $(ARCH_DIR)/include
 ccflags-y		+= -iquote .
-ccflags-y		+= -I/usr/include/libnl3
 ccflags-y		+= $(COMPEL_UAPI_INCLUDES)
 ccflags-y		+= $(CONFIG-DEFINES)
 
-- 
2.14.1

