From 453023690c48e51263a04ab365cc0073d90d41ca Mon Sep 17 00:00:00 2001
From: "A.I" <ailis@paw.zone>
Date: Thu, 10 Mar 2016 14:53:52 +0900
Subject: [PATCH 1/2] Remove quotes around $(CC) for buildroot

---
 criu/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/criu/Makefile b/criu/Makefile
index b696c63..f2fa9b4 100644
--- a/criu/Makefile
+++ b/criu/Makefile
@@ -89,7 +89,7 @@ ifneq ($(MAKECMDGOALS),mrproper)
         ifneq ($(shell sh -c                                                            \
                         'TMP="$(OUTPUT)$(TMPOUT).$$$$";                                 \
                         echo "int main(int argc, char *argv[]) { return 0; }" |         \
-                        "$(CC)" -x c - $(LIBS) -o "$$TMP" > /dev/null 2>&1 && echo y;   \
+                        $(CC) -x c - $(LIBS) -o "$$TMP" > /dev/null 2>&1 && echo y;   \
                         rm -f "$$TMP"'),y)
                 $(warning "Couldn't find some of the required libraries")
                 $(warning "Make sure the following packages are installed")
-- 
2.5.4 (Apple Git-61)

